commands/koth_quotes.py

```diff
@@ -0,0 +1,50 @@
+from commands.base import BaseCommand
+from random import randint
+
+class KOTHQuotes(BaseCommand):
+    trigger = 'quote'
```

Comment:
Personally, I think the trigger should be `koth-quote`, `kothquote`. Either that or we make this more extensible by having a `quote` command that can give quotes from different shows/movies/games. Then we could do `quote koth` to get a King of the Hill quote, and `quote <another show>` for other ones

Reply:
I agree. I like the second idea because it won't limit ourselves when we decide to change the bot from Hank Hill to, say, Bojack Horseman

Reply:
Perhaps we just leave it as is for this PR and make a task to genericize it for later implementation?

Reply:
I like the Idea of leaving it for now, but generalizing it later.

Reply:
Resolving with "make a task to make it generic later"